src/main/java/com/leetcode/algorithm/easy/SameTree.java

```diff
@@ -0,0 +1,19 @@
+package com.leetcode.algorithm.easy;
+
+/**
+ * https://leetcode.com/problems/same-tree/
```

Comment:
Please add the following information. This is automatically generate the README file. 

Thanks. 

 * 100. Same Tree
 * Difficulty : Easy
 * Tags : Tree
* 
* https://leetcode.com/problems/same-tree/